src/sinks/pulsar.rs

```diff
@@ -356,9 +357,13 @@ impl Sink<Event> for PulsarSink {
                         protocol: "tcp",
                     });
                 }
-                Some((Err(error), _, finalizers)) => {
+                Some((Err(error), metadata, finalizers)) => {
                     finalizers.update_status(EventStatus::Errored);
                     error!(message = "Pulsar sink generated an error.", %error);
```

Comment:
It seems like we should drop this `error!` since the event is also emitting a log.

Reply:
Good catch!